index.html

```diff
@@ -1,657 +1,1213 @@
-                            <h3>Changing the World Takes Leaders Like You</h3>
-                        </div>
-                    </div>
-                    <div class="text">
-                        <div data-cmp-data-layer="" id="" class="text-wrap">
-                            <p>When leaders are transformed, we affect change through the channels and connections they have – family, friends, co-workers and colleagues. Deep, positive, long-lasting change that can help make our world better.</p>
-                        </div>
+                  </div>
+
+                  <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="title-c6c8e598f8" class="cmp-title">
+                      <h3 class="cmp-title__text">Changing the World Takes Leaders Like You</h3>
                     </div>
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">More About Us</span>
-                        </a>
+                  </div>
+
+                  <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="text-924f8b967f" class="cmp-text">
+                      <p>When leaders are transformed, we affect change through the channels and connections they have – family, friends, co-workers and colleagues. Deep, positive, long-lasting change that can help make our world better.</p>
                     </div>
+                  </div>
+
+                  <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <a id="button-254edd7bf9" class="cmp-button" href="/content/leaderimpact/lang/en2/stories.html">
+                      <span class="cmp-button__text">More About Us</span>
+                    </a>
+                  </div>
                 </div>
-                <div class="icon-box-wrap">
-                    <div class="icon-box-item">
-                        <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                            <img srcset="./assets/article-item.svg" src="./assets/article-item.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
+              </div>
+              <div class="container responsivegrid icon-box-wrap aem-GridColumn aem-GridColumn--default--6 layout-grid">
+                <div id="container-85b30b8ad8" class="cmp-container">
+                  <div class="teaser icon-box-item-col">
+                    <div id="teaser-66129ff9d4" class="cmp-teaser" data-cmp-data-layer="{&quot;teaser-66129ff9d4&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/teaser&quot;,&quot;dc:title&quot;:&quot;Teaser Title&quot;,&quot;dc:description&quot;:&quot;<p>Teaser Description</p>&quot;}}">
+                      <div class="cmp-teaser__image image">

```

Comment:
@Prystal254 We can't change the order of the component output. Please review the teaser component [documentation](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/teaser.html#tabs-0952fef056-item-765c2c16ef-tab).